src/components/ICOApp.js

```diff
@@ -4,12 +4,15 @@ import { getValueOrDefault, icoTransparencyMap, trimString } from '../utils';
 import { connect } from 'react-redux';
 import { default as config } from '../config.js';
 
+const midifyUrl = (url) => {
```

Comment:
`midifyUrl` has a typo, also it does not say what it really does, should be smth like `extractHostnameFromUrl`

Reply:
👍